legal-api/src/legal_api/models/business.py

```diff
@@ -298,10 +298,7 @@ def business_legal_name(self):
 
         Legal Name Easy Fix
         """
-        from legal_api.services import flags  # pylint: disable=import-outside-toplevel
-
-        flag_on = flags.is_on('enable-legal-name-fix')
```

Comment:
Now that we released the easy legal name fix, it's a good idea to clean up the FF everywhere from the code as per Kial's comment.